pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -144,7 +144,7 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod) (
 		CgroupParent: cgroupParent,
 		SecurityContext: &runtimeapi.LinuxSandboxSecurityContext{
 			Privileged:         kubecontainer.HasPrivilegedContainer(pod),
-			SeccompProfilePath: m.getSeccompProfileFromAnnotations(pod.Annotations, ""),
+			SeccompProfilePath: m.getSeccompProfile(pod.Annotations, "", nil, nil),
```

Comment:
why are we not passing in the security context fields here?

Reply:
Well spotted. I am now only setting the PodSecurityContext, as that is analogous to the existing behaviour.

Reply:
I created some tests around this to check that container-level seccomp field and annotation are ignored. 

Currently, there is no way to set seccomp to a sandbox container at container-level. Am I right to assume we are keeping this behaviour? @tallclair @liggitt